validate/README.md

```diff
@@ -0,0 +1,11 @@
+<!-- cargo-sync-readme start -->
+
+Constants and methods for validating request parameters.
+
+This crate is used internally by [`twilight-http`], but may also be used for
+manually validating any models from [`twilight-model`].
+
+[`twilight-http`]: https://docs.rs/twilight-http/latest
+[`twilight-model`]: https://docs.rs/twilight-model/latest
+
+<!-- cargo-sync-readme end -->
```

Comment:
The style of this README is not consistent with other crates (missing top-level title, badges, ...)